src/lib.rs

```diff
@@ -723,6 +722,11 @@ mod tests {
         error_policy_test!("module_cycle.cas", 1, ErrorItem::Compile(_));
     }
 
+    #[test]
+    fn invalid_system_error() {
+        error_policy_test!("system_invalid.cas", 5, ErrorItem::Compile(_));
```

Comment:
Can we add invalid tests for:

1. Missing a required config
2. Invalid module
3. Same config multiple times